.github/paths-labeller.yml

```diff
@@ -0,0 +1,53 @@
+---
+- "beats":
+  - "detection_rules/beats.py"
+  - "etc/beats_schemas/**/*"
+- "documentation":
+  - "./**/*.md"
```

Comment:
does this also match `.md` in the root folder?

Reply:
it should per glob rules

```python
glob.glob('detection-rules/**/*.md', recursive=True)
detection-rules-fork/CLI.md
detection-rules-fork/README.md
detection-rules-fork/CONTRIBUTING.md
detection-rules-fork/PHILOSOPHY.md
detection-rules-fork/rta/README.md
detection-rules-fork/docs/rule_insights.md
detection-rules-fork/docs/deprecating.md
detection-rules-fork/docs/experimental-machine-learning/DGA.md
detection-rules-fork/docs/experimental-machine-learning/problem-child.md
detection-rules-fork/docs/experimental-machine-learning/host-risk-score.md
detection-rules-fork/docs/experimental-machine-learning/url-spoof.md
detection-rules-fork/docs/experimental-machine-learning/readme.md
detection-rules-fork/docs/experimental-machine-learning/experimental-detections.md
detection-rules-fork/rules/README.md
```